.pre-commit-config.yaml

```
@@ -16,12 +16,12 @@ repos:
     -   id: auto-walrus
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.0.278
+    rev: v0.0.277
```

```suggestion
    rev: v0.0.280
```
And then disable individual rules in `pyproject.toml`.